src/components/show-products/index.js

```diff
@@ -107,7 +115,7 @@ export const ShowProducts = () => {
         {searchTerm && (
           <input
             className="close-icon"
-            onClick={deleteSearchTerm}
+            onClick={() => setSearchTerm('')}
```

Comment:
I think it is better this way, as it was just a one-line function which could've been easily used within the onClick event just the way you did right now.